plugins/modules/foreman_template_sync.py

```diff
@@ -143,12 +143,12 @@
 
 RETURN = ''' # '''
 
-from ansible.module_utils.foreman_helper import ForemanEntityAnsibleModule
+from ansible.module_utils.foreman_helper import ForemanAnsibleModule
 
 
 def main():
-    module = ForemanEntityAnsibleModule(
-        entity_spec=dict(
+    module = ForemanAnsibleModule(
+        argument_spec=dict(
             direction=dict(choices=['import', 'export'], default='import'),
             location=dict(type='entity', flat_name='location_id'),
```

Comment:
With `argument_spec` you probably loose all the nice additions from our `entity_spec`. :crab: 
We had that discussion earlier. Maybe it's time to revisit.
@evgeni @bagasse ?

Reply:
I thought entity_spec won't work with ForemanAnsibleModule

Reply:
Right. But without entity_spec you can't use the extensions we have (like type:entity, and flat_name etc).
Which leads us to https://github.com/theforeman/foreman-ansible-modules/issues/404 ;)